src/pages/instances/Events.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, useEffect, useState } from "react";
+import { useInstanceLoading } from "context/instanceLoading";
+import { LxdEvent } from "types/event";
+
+const Events: FC = () => {
+  const instanceLoading = useInstanceLoading();
+  const [eventWs, setEventWs] = useState<WebSocket | null>(null);
+
+  const handleEvent = (event: LxdEvent) => {
+    if (event.metadata.action === "instance-created") {
+      const instanceName = event.metadata.source.split("/").pop() ?? "";
+      const entry = instanceLoading.get(instanceName);
+      if (entry && entry.type === "Creating" && entry.callback) {
+        entry.callback();
+      }
+    }
+  };
```

Comment:
The idea is this will handle multiple things in the future. I just added the error case as well. We probably need to organize it a bit better when handling more types, but it should be generic for all types of events.